metadata-ingestion/examples/ai/vertexai_example_experiment.py

```diff
@@ -0,0 +1,141 @@
+import argparse
+
+from dh_ai_client import DatahubAIClient
+
+import datahub.metadata.schema_classes as models
+from datahub.metadata.com.linkedin.pegasus2avro.dataprocess import RunResultType
+
+
+def create_experiment_example(client: DatahubAIClient) -> None:
+    experiment_urn = client.create_experiment(
+        experiment_id="table_classification_experiment",
+        properties=models.ContainerPropertiesClass(
+            name="Tabular classification Experiment",
+            description="Experiment for tabular classification",
+            customProperties={"team": "forecasting"},
+            created=models.TimeStampClass(
+                time=1628580000000, actor="urn:li:corpuser:datahub"
+            ),
+            lastModified=models.TimeStampClass(
+                time=1628580000000, actor="urn:li:corpuser:datahub"
+            ),
+        ),
+    )
+
+    # Create a training run
+    run_urn = client.create_training_run(
+        run_id="simple_training_run",
+        properties=models.DataProcessInstancePropertiesClass(
+            name="Simple Training Run",
+            created=models.AuditStampClass(
+                time=1628580000000, actor="urn:li:corpuser:datahub"
+            ),
+            customProperties={"team": "forecasting"},
+        ),
+        training_run_properties=models.MLTrainingRunPropertiesClass(
+            id="simple_training_run",
+            outputUrls=["gc://my-bucket/output"],
+            trainingMetrics=[models.MLMetricClass(name="accuracy", value="0.9")],
+            hyperParams=[models.MLHyperParamClass(name="learning_rate", value="0.01")],
+            externalUrl="https:localhost:5000",
+        ),
+        run_result=RunResultType.FAILURE,
+        start_timestamp=1628580000000,
+        end_timestamp=1628580001000,
+    )
+
+    # Create model group
```

Comment:
not really sure the model-related code belongs in this script ; wouldn't it be more straightforward if we do this since job - model is directly related?
* example_experiment.py: experiment - run
* exmple_model.py : job - model  - model group